docs/bundler/executables.md

```diff
@@ -45,6 +45,58 @@ When deploying to production, we recommend the following:
 bun build --compile --minify --sourcemap ./path/to/my/app.ts --outfile myapp
 ```
 
+### Cross-compile to other platforms
+
+To target a different platform, architecture, or version of Bun, use the `--target` flag.
+
+To build for Linux from macOS, Windows or Linux, you can use the following command:
+
+```sh
+# The default architecture is x64 if an os is specified.
+bun build --compile --target=bun-linux ./index.ts --outfile myapp
```

Comment:
Feels like we should make it `--target=linux` vs `--target=bun-linux`. Less words, also bun is obvious.